### ISSUE TITLE
Yocto Version

### ISSUE THREAD

**Issue by telphan** [siemens/meta-iot2000#60]
Any reason behind this layer not supporting the pyro yocto version?

**Comment by jan-kiszka** [comment]
Yes: breakages around the Galileo/Arduino recipes we inherited from the original Intel BSP, specifically around uclibc. Unfortunately, we still need to solving those bugs or update the Arduino toolchain before we can update.
That said, the bsp layer should work with Pyro. If not, let us know, that can and should be fixed earlier.

**Comment by jan-kiszka** [comment]
Check out https://github.com/siemens/meta-iot2000/tree/jan/rocko.

**Comment by jan-kiszka** [comment]
Merged into master now.
